Tests/Framework/Audio/SoundEffectInstanceTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.IO;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Audio;
+using NUnit.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace MonoGame.Tests.Audio
+{
+    class SoundEffectInstanceTest
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            // Necessary to get audio initialised
+            FrameworkDispatcher.Update();
+        }
+
+        /// <summary>
+        /// Unit test for issue #7372 where the Sound effects instance does not play after Play()
+        /// is called after calling Pause(), Stop().
+        /// </summary>
+        [Test]
+#if !DESKTOPGL
```

Comment:
No need to exclude this for other platforms, so this directive can be removed.

Reply:
I originally intended to make it a test for both platforms but when I tried testing the DirectX Audio, its implementation sets the state to stop rather than pause. 
I noticed that after additional testing that the Assert.AreEqual(SoundState.Stopped, instance) is runs to quickly after calling stop. This can be temporarily fixed by making the thread sleep for 10 ms before running the test, but it may be an indicator of additional bugs.